public/js/script.js

```diff
@@ -131,8 +148,9 @@ $(document).ready(function() {
 });
 function upvote(id, number) {
   console.log('upvote clicked');
-  socket.emit('upvote', {id:id, number:number});
+  socket.emit('upvote', {id:id, number:parseInt(number)});
```

Comment:
Because we now use triple equals instead of double equals, we need to make sure types match. The number in the server is an int, so we need to convert this string to an int.